### ISSUE TITLE
Validation dataloder for masking experiments

### ISSUE THREAD

**Issue by ankitpatnala** [ecmwf/WeatherGenerator#393]
### What happened?

I just noticed in the masking pre-training experiments, we have masked value=0.6. For the validation, as it take default config from the json file in the respective model directory, it also does masking in the inference run.

Should not it be overridden with masking_rate=0.0, so that we make consistent test with different validation runs? We can definitely overwrite with the overwrite config file but do we ever need masking in validation set?

### What are the steps to reproduce the bug?

Simply run pre-training from training_mode=masking
Then use inference from same run_id

Then check number of files corresponding to each time stamp in the generated zarr file
 

### Version

0.1

### Platform (OS and architecture)

Juwels Booster (linux)

### Relevant log output

```shell

```

### Organisation

JSC

**Comment by clessig** [comment]
The current behavior is as designed. When one does evaluation runs, then one will, without changes to the config, test the skill of the masking. There are many instances where one will change the masking strategy for evaluation (we have done this a lot for atmorep) but then the config should be explicitly overwritten.
